spec/controllers/api/lists_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe Api::ListsController do
+  describe "POST create" do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
```

Comment:
Line is too long. [139/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.